### ISSUE TITLE
Missing info on fixing: Using events for service checks is deprecated in favor of monitors

### ISSUE THREAD

**Issue by ssbarnea** [DataDog/dd-agent#1128]
It seems that you deprecated a feature without putting any information regarding how to fix it. I even googled and the only reference is the source code where you added this warning.

Please provide some information regarding what changes we are supposed to implement in order not to get these warnings. 

These warnings are showing in the UI:
https://www.dropbox.com/s/xpjpmgqtk71hqze/Screenshot%202014-09-24%2015.43.22.png?dl=0

**Comment by ssbarnea** [comment]
I am still hoping to see this problem resolved. Running Agent version 5.1.1 and still getting this unclear error.

**Comment by rmoriz** [comment]
same here with dd-agent 5.2.1. Pollutes our dd-webviews :-(

**Comment by ineu** [comment]
@remh Still the case with 
```
-bash-4.2# rpm -qa | grep datadog
datadog-agent-5.3.0-1.x86_64
```

**Comment by remh** [comment]
@ineu you should turn the
```yaml
skip_event
```
flag to True:
https://github.com/DataDog/dd-agent/blob/master/conf.d/http_check.yaml.example#L86-L90

and make sure you've setup a service check on the web app.

**Comment by ineu** [comment]
@remh It works, thank you.

**Comment by hron84** [comment]
We getting same warning for the default `system` check. Any update on this? We do not want to turn off `skip_event` on system checks even if it is possible.

**Comment by anandkumarpatel** [comment]
+1 getting this on tcp_check 
agent version (v 5.5.1)
```
    tcp_check
    ---------
      - instance #0 [WARNING]
          Warning: Using events for service checks is deprecated in favor of monitors and will be removed in future versions of the Datadog Agent.
      - instance #1 [WARNING]
          Warning: Using events for service checks is deprecated in favor of monitors and will be removed in future versions of the Datadog Agent.
      - instance #2 [WARNING]
          Warning: Using events for service checks is deprecated in favor of monitors and will be removed in future versions of the Datadog Agent.
      - instance #3 [WARNING]
          Warning: Using events for service checks is deprecated in favor of monitors and will be removed in future versions of the Datadog Agent.
      - Collected 4 metrics, 0 events & 5 service checks
```

**Comment by ssbarnea** [comment]
I do have skip_event: True for all instances inside the http_check.yml and I still see the warning! Even with Collector (v 5.6.3)

**Comment by remh** [comment]
@ssbarnea can you share your config ?
I just tried to reproduce but i don't get any warnings when `skip_event: True`

**Comment by anroots** [comment]
Defining `skip_event: true` in `init_config` seems to have no effect. Defining it as an instance setting works (no deprecation warnings).

**This will output warnings**
```yaml
init_config:
  skip_event: true
instances:
  - name: sqroot.eu
    url: https://sqroot.eu
```

**This will not**
```yaml
init_config:
instances:
  - name: sqroot.eu
    skip_event: true
    url: https://sqroot.eu
```

Ideally, I'd like to define this parameter once, globally.

I'm running Dockerized `Collector (v 5.7.3)`.

**Comment by ava-dylang** [comment]
From the [http_check.yaml.example](https://github.com/DataDog/dd-agent/blob/master/conf.d/http_check.yaml.example)
```
    # The (optional) skip_event parameter will instruct the check to not
    # create any event to avoid duplicates with a server side service check.
    # This default to False.
    #
    skip_event: true
```
I don't understand what this flag controls, it doesn't sound like it 's designed to squelch the dd.collector http_check warnings, more like it disables the machinery behind the warning altogether. I neither assume what it means about it duplicating "server side checks", datadog _is running on the server side_, what other server would duplicate the first server's checks?

My team uses datadog checks for alerting purposes,
What I really would like to know is, would disabling events (enabling skip_events) implicitly disable posting check statuses to datadog? Or is disabling events a safe change for http_checks?

**Comment by eBeyond** [comment]
I had the same issue. BUT as ssbarnea mentioned 'skip_event: True' removes the warning.
But the issue here is: 'skip_event: true' doesn't do it.
You have to have an uppercase 'T'

**Comment by ava-dylang** [comment]
`skip_event: true` (lowercase t) is working for me with datadog agent 5.10.1 on python 2.7.12. I do have to set that property on every single http_check instance though.

**Comment by stetskevych** [comment]
It's 2017, and I still get this warning :+1:

**Comment by codedge** [comment]
With `skip_events: true` there is no warning showing up. I am otally fine with this.
Of course a general configution in `init_config` would be better.

**Comment by mpalermo73** [comment]
Here we are in **2018** now and _still_ getting the same error!  WOOOOO!

`2018-01-03 18:31:28 UTC | WARNING | dd.collector | checks.http_check(__init__.py:693) | Using events for service checks is deprecated in favor of monitors and will be removed in future versions of the Datadog Agent.`

M.

**Comment by ava-dylang** [comment]
Was there a clear upgrading documented for the changes required to move off http_checks to monitors? I haven't stumbled into such document yet :bowing_man:

**Comment by hron84** [comment]
@zippolyte Did you fixed this issue? Which version will contain the fix?

**Comment by zippolyte** [comment]
@hron84 Yes the two PRs fix the issue. They will be released with agent 5.23, scheduled for the end of next week.

**Comment by hron84** [comment]
@zippolyte Thanks
